### PR TITLE
Also broadcast if the post wasn't found

### PR DESCRIPTION
**Pull request by cguess**
Welp, this fixes an issue where if the post wasn't found then the scrape was never broadcasted.

Whoops

### CODE REVIEW

**Review by reefdog — APPROVED** [review]
I trust this solves something, lol.

**Review by reefdog — CHANGES_REQUESTED** [review]
Actually, revoking my approval, because I actually ran tests and am consistently getting this error (not on `master`):

```
Error:
MediaVault::IngestControllerTest#test_Submitting_real_JSON_but_with_a_bad_schemas_to_the_ingest_API_gives_a_400:
NoMethodError: undefined method `[]' for nil:NilClass

    appearance = media_review["itemReviewed"]["mediaItemAppearance"].select do |appearance|
                                             ^^^^^^^^^^^^^^^^^^^^^^^
    app/models/archive_item.rb:24:in `create_from_media_review'
    app/controllers/media_vault/ingest_controller.rb:164:in `archive_from_media_review'
    app/controllers/media_vault/ingest_controller.rb:79:in `submit_media_review'
    test/controllers/media_vault/ingest_controller_test.rb:21:in `block in <class:IngestControllerTest>'

rails test test/controllers/media_vault/ingest_controller_test.rb:20
```

Can you take a look @cguess and see if it's legit, or a broken test, or what?

File: app/controllers/media_vault/ingest_controller.rb [hunk 1]
```diff
@@ -55,6 +67,15 @@ def message
       else T.absurd(self)
       end
     end
+
+    sig { returns(StandardError) }
+    def error
+      case self
+      when JSONParseError then JSONParseException.new
+      when JSONValidationError then JSONValidationException.new
+      else T.absurd(self)
```
Comment (reefdog) [inline app/controllers/media_vault/ingest_controller.rb:76]: `T.absurd` 😆, I can't believe it's [real](https://sorbet.org/docs/exhaustiveness)!

File: app/controllers/media_vault/ingest_controller.rb [hunk 2]
```diff
@@ -86,9 +110,17 @@ def submit_media_review
       failures: media_review_json
     }
     render(json: response_payload, status: 400)
+  rescue ApiErrors::JSONValidationException
+    response_payload = {
+      error_code: ApiErrors::JSONValidationError.code,
+      error: ApiErrors::JSONValidationError.message,
+      failures: media_review_json
+    }
+    render(json: response_payload, status: 400)
```
Comment (reefdog) [inline app/controllers/media_vault/ingest_controller.rb:119]: Total side note (as you are correctly being consistent w/the rest of this file), but I've been preferring the [keywords](https://guides.rubyonrails.org/layouts_and_rendering.html#options-for-render) myself, just to save us a lookup to the cheat sheet.

No change requested of course, just raising the point.

File: app/controllers/media_vault/ingest_controller.rb [hunk 3]
```diff
@@ -76,6 +97,9 @@ def submit_media_review
     # TODO: Spin off an active job to handle this
     typed_params = TypedParams[SubmitMediaReviewParams].new.extract!(params)
     media_review_json = JSON.parse(typed_params.media_review_json)
```
Comment (reefdog) [inline app/controllers/media_vault/ingest_controller.rb:99]: Should this try to catch `JSONParseError` or no? Just making sure I understand the point of it.

File: config/database.yml [hunk 4]
```diff
@@ -84,4 +84,6 @@ test:
 # for a full overview on how database connection configuration can be specified.
 #
 production:
-  url: <%= ENV["DATABASE_URL"] %>
+  <<: *default
+  database: zenodotus_production
+  # url: <%= ENV["DATABASE_URL"] %>
```
Comment (reefdog) [inline config/database.yml:89]: I don't think this change was intended to be committed, was it?

File: app/controllers/media_vault/ingest_controller.rb (hunk 3 above)
Comment (reefdog) [reply app/controllers/media_vault/ingest_controller.rb:99]: 🤦 Apologies, I forgot to expand GitHub's interface to see the whole context.

**Review by reefdog — CHANGES_REQUESTED** [review]
Thanks for fixing that test! I'm getting a new failing test, and it also looks like you accidentally checked in a change to the production database configuration.

Here's the test that's failing for me now:
```
Error:
MediaVault::IngestControllerTest#test_Can_archive_media_review_from_json:
ActiveRecord::RecordInvalid: Validation failed: Media authenticity category can't be blank
    app/models/archive_item.rb:40:in `create_from_media_review'
    app/controllers/media_vault/ingest_controller.rb:202:in `archive_from_media_review'
    app/controllers/media_vault/ingest_controller.rb:103:in `submit_media_review'
    test/controllers/media_vault/ingest_controller_test.rb:88:in `block in <class:IngestControllerTest>'

rails test test/controllers/media_vault/ingest_controller_test.rb:87
```

I've verified it's not breaking on `master`.

It also looks like you committed a regression to the schema version? Though it's not showing up in the "Files changed" tab, commit 9b4fa08d5399c322f916e80fdc849b77704ea418 shows the regression and is also reflected in the local code when I pull, which means running `rails db:migrate` locally creates a change (updating the schema version).

File: config/database.yml (hunk 4 above)
Comment (cguess) [reply config/database.yml:89]: damn it, fixing now

**Review by reefdog — APPROVED** [review]
Tests run clean, diff looks good! 👍

**Pull request merged**